src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx

```diff
@@ -449,23 +450,25 @@ const AwardAmountsChart = ({
                     text: 'Obligated Amount',
                     color: obligatedColor
                 },
-                numerator2: {
-                    className: awardAmounts._nonFederalFunding > 0 ? `asst-non-federal-funding` : `asst-nff-zero`,
-                    labelSortOrder: 1,
-                    labelPosition: 'bottom',
-                    // fudging this for to get the correct tooltip position.
-                    rawValue: awardAmounts._nonFederalFunding + awardAmounts._totalObligation,
-                    denominatorValue: awardAmounts._totalFunding,
-                    lineOffset: lineOffsetsBySpendingCategory.nonFederalFunding,
-                    barWidthOverrides: {
-                        applyToLine: true,
-                        rawValue: awardAmounts._nonFederalFunding,
-                        denominatorValue: awardAmounts._totalFunding
-                    },
-                    value: awardAmounts.nonFederalFundingAbbreviated,
-                    color: nonFederalFundingColor,
-                    text: "Non Federal Funding",
-                    tooltipData: getTooltipPropsByAwardTypeAndSpendingCategory(awardType, 'nonFederalFunding', awardAmounts)
+                ...isNffZero ? {} : {
```

Comment:
this is the right idea but we can't do a [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) on a variable whose value is a boolean